pyramid_zipkin/logging_helper.py

```diff
@@ -11,8 +11,17 @@
 from pyramid_zipkin.thrift_helper import create_span
 from pyramid_zipkin.thrift_helper import thrift_obj_in_bytes
 
+try:  # Python 2.7+
```

Comment:
My opinion is to drop py26 as well, but it is specified as supported in tox.ini and this particular change is easy enough. Perhaps we should open an issue to track removing py26 support?